src/index.html

```diff
@@ -5,8 +5,279 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img class="header__image" src="./images/logo.png"
+        alt="icon">
+      </a>
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__list">
+            <a href="#apple" class="is-active nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#smartphonesAndTelephones" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#laptopsAndCompurest" class="nav__link"
+            data-qa="hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#gadgets" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#tablets" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#photo" class="nav__link">
+              Фото
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#video" class="nav__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container" data-qa="card">
+        <img src="./images/imac.jpeg" alt="imac"
+         class="main__image">
+        <h2 class="main__headline">Моноблок APPLE A1419 iMac 27&quot;  Retina 5K (MNED2UA/A)</h2>
+        <p class="main__code">Код товара: 195434</p>
+        <div class="main__revies">
+          <div class="main__revies stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="main__count-review">Отзывов: 5</p>
+        </div>
+        <div class="main__cost-info">
+          <p class="main__cost">Цена:</p>
+          <p class="main__how-cost">69 999 грн</p>
+        </div>
+        <div class="main__button">
+          <a href="#buy" data-qa="hover"
+           class="call-back-button">
```

Comment:
```suggestion
          <a 
            href="#buy" 
            data-qa="card-hover"
            class="call-back-button"
          >
```
![image](https://user-images.githubusercontent.com/102380127/179348967-f5c88855-e63f-4ad5-a8d5-551a190a0517.png)